core/pfe-core/controllers/roving-tabindex-controller.ts

```diff
@@ -188,17 +191,28 @@ export class RovingTabindexController implements ReactiveController {
   /**
    * from array of HTML items, and sets active items
    */
-  initItems(items: HTMLElement[]) {
+  initItems(items: HTMLElement[], itemsContainer?: HTMLElement) {
     this.#items = items ?? [];
     const focusableItems = this.#focusableItems;
     const [focusableItem] = focusableItems;
     this.#activeItem = focusableItem;
     for (const item of focusableItems) {
       item.tabIndex = this.#activeItem === item ? 0 : -1;
     }
+
+    /**
+     * removes listener on previous contained and applies it to new container
+     */
+    if (!this.#itemsContainer || itemsContainer !== this.#itemsContainer) {
+      if (this.#itemsContainer) {
+        this.#itemsContainer.removeEventListener('keydown', this.#onKeydown.bind(this));
+      }
+      this.#itemsContainer = itemsContainer || this.host;
+      this.#itemsContainer.addEventListener('keydown', this.#onKeydown.bind(this));
+    }
   }
 
   hostConnected() {
-    this.host.addEventListener('keydown', this.#onKeydown.bind(this));
+    this.#itemsContainer = undefined;
```

Comment:
this will blow away the state

```js
const el = document.createElement('has-rti-controller');
$('#container').append(el); // ok
el.rtiController.initItems(...items); // alright
$('#other-container').append(el); // oops... hostConnected()
```